django/db/models.py

```diff
@@ -226,18 +222,19 @@ class FieldInformation(models.Model):
     manager                 = FieldsManager()
     objects                 = models.Manager() # default manager
     
-    table                   = _CHAR(max_length=35, **_NULLOKSTR)
-    field                   = _CHAR(max_length=35, **_NULLOKSTR)
-    alias                   = _CHAR(max_length=35, **_NULLOKSTR)
-    queryset                = _CHAR(max_length=35, **_NULLOKSTR)
+    table                   = _TEXT(**_NULLOKSTR)
```

Comment:
I think a text field is basically the same thing as a varchar with no length limit, internally.  see http://www.depesz.com/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/, for instance, for commentary.